doc/manpage-urls.json

```diff
@@ -146,6 +146,9 @@
   "bootup(7)": "https://www.freedesktop.org/software/systemd/man/bootup.html",
   "daemon(7)": "https://www.freedesktop.org/software/systemd/man/daemon.html",
   "file-hierarchy(7)": "https://www.freedesktop.org/software/systemd/man/file-hierarchy.html",
+  "foot.ini(1)": "https://man.archlinux.org/man/foot.1.en",
+  "footclient(1)": "https://man.archlinux.org/man/footclient.1.en",
```

Comment:
Should unused manuals be added or not?

Reply:
Doesn't really hurt and probably makes updating the module docs less hazardous.